byterun/minor_gc.c

```diff
@@ -372,6 +372,9 @@ static void oldify_one (void* st_v, value v, value *p)
 }
 
 /* Test if the ephemeron is alive */
+/* Care needed with this test. It will only make sense if
+   all minor heaps have been promoted
+   as the liveness of the keys can only be known at that point */
 static inline int ephe_check_alive_data (struct caml_ephe_ref_elt *re,
```

Comment:
It seems like we don't use the `young_ptr` and `young_end` here, could we remove them from the signature?

Reply:
Done!